### PR TITLE
[Middleware_01] Create point class

### PR DESCRIPTION
**Pull request by andrewdimmer**
Working on #24 - Wrote the test class and code

### CODE REVIEW

File: source/edu/oakand/test/middleware01/PointTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.oakland.helper.middleware01.Point;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("JUnit 5 Point Test")
+public class PointTest {
+
+  private int coordX = (int) (Math.random() * 100);
```
Comment (NathanDimmer) [inline source/edu/oakand/test/middleware01/PointTest.java:12]: Maybe this should not be random, don't want it the same as y.

File: source/edu/oakand/helper/middleware01/Point.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package edu.oakland.helper.middleware01;
+
+public class Point {
+
+  public int coordX;
+  public int coordY;
```
Comment (NathanDimmer) [inline source/edu/oakand/helper/middleware01/Point.java:6]: These should be private

File: source/edu/oakand/helper/middleware01/Point.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  public Point(int coordX, int coordY) {
+    this.coordX = coordX;
+    this.coordY = coordY;
+  }
```
Comment (NathanDimmer) [inline source/edu/oakand/helper/middleware01/Point.java:11]:
```suggestion
    public int getX() {
      return coordX;
    }
    
    public int getY() {
      return coordY;
    }
  }
```

File: source/edu/oakand/test/middleware01/PointTest.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  private int coordY = (int) (Math.random() * 100);
+
+  @Test
+  @DisplayName("coordX stored correctly")
+  void coordxEqual() {
+    Point p = new Point(coordX, coordY);
+    assertEquals(p.coordX, coordX);
```
Comment (NathanDimmer) [inline source/edu/oakand/test/middleware01/PointTest.java:19]:
```suggestion
    assertEquals(p.getX(), coordX);
```

File: source/edu/oakand/test/middleware01/PointTest.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  @Test
+  @DisplayName("coordY stored correctly")
+  void coordyEqual() {
+    Point p = new Point(coordX, coordY);
+    assertEquals(p.coordY, coordY);
```
Comment (NathanDimmer) [inline source/edu/oakand/test/middleware01/PointTest.java:26]:
```suggestion
    assertEquals(p.getY(), coordY);
```

**Review by NathanDimmer — CHANGES_REQUESTED** [review]
Just a few changes, then good to go.

File: source/edu/oakand/helper/middleware01/Point.java (hunk 2 above)
Comment (andrewdimmer) [reply source/edu/oakand/helper/middleware01/Point.java:6]: Yep, will fix.

File: source/edu/oakand/test/middleware01/PointTest.java (hunk 1 above)
Comment (andrewdimmer) [reply source/edu/oakand/test/middleware01/PointTest.java:12]: Random works well enough.

**Review by NathanDimmer — APPROVED** [review]
Looks good!

**Pull request merged**